File: Gradle/Java/MapStruct-Lombok/src/main/java/com/qbros/gradlejavamapstructlombok/Controller.java
package com.qbros.gradlejavamapstructlombok;

import lombok.RequiredArgsConstructor;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import static com.qbros.gradlejavamapstructlombok.Controller.AccountControllerURI;


@RestController
@RequiredArgsConstructor
@RequestMapping(AccountControllerURI)
public class Controller {

    static final String AccountControllerURI = "/accounts";
    private final AccountMapper accountMapper;

    @GetMapping
    public AccountDto getAccount() {

        AccountModel accountModel = new AccountModel(123, 12.3f, "dfdf");
        return accountMapper.toDto(accountModel);
    }
}

File: Gradle/Kotlin/MapStruct-Klog/src/main/kotlin/com/qbros/gradlejavamapstructlombok/Controller.kt
package com.qbros.gradlejavamapstructlombok

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

const val AccountControllerURI = "/accounts"

@RestController
@RequestMapping(AccountControllerURI)
class Controller(private val accountMapper: AccountMapper) {

    @GetMapping
    fun getAccount(): AccountDto {
        val accountModel = AccountModel(123, 12.5f, "abc");
        return accountMapper.toDto(accountModel)
    }
}